micrometer-observation/src/main/java/io/micrometer/observation/aop/ObservedAspect.java

```diff
@@ -178,7 +178,7 @@ private void stopObservation(Observation observation, Observation.Scope scope, @
         observation.stop();
     }
 
-    public static class ObservedAspectContext extends Observation.Context {
+    static class ObservedAspectContext extends Observation.Context {
```

Comment:
If we make this not public, then users can't make an `ObservationConvention` for `ObservedAspectContext`. Seeing as the public constructor takes an `ObservationConvention<ObservedAspectContext>`, I assume we want to allow users to do that.

Reply:
You are right, fixed.